cli/cluster/generate.go

```diff
@@ -0,0 +1,3 @@
+package cluster
+
+//go:generate ./env_generate.lua
```

Comment:
Is generation supposed to be invoked manually?

Reply:
At the moment yes.

Doing it automatically before every tt build doesn't seem like a good idea (because the result depends on a Tarantool version). Maybe add an extra target to the magefile for a manual run?

Reply:
Yes, I understand that it is not a good idea to run it on each mage build. I meant... What should I run to update the file? What command in which directory?

Reply:
1. Install a latest Tarantool.
2. go generate ./... in the root folder of `tt`.

Reply:
I've added the instruction into the generated file:

```Go
// To update the file:
// 1. Install a latest Tarantool version.
// 2. Run: go generate ./...
```

Reply:
I've added `mage generate` target.